### DISCUSSION TITLE
Json schema for custom block

### DISCUSSION THREAD

**Discussion by kingyue737** [vuejs/language-tools#1503] [Ideas]
I'm using [vite-plugin-page](https://github.com/hannoeru/vite-plugin-pages) which allowed to define vue route config with a custom json block. I wonder is it possible to map custom block to a json schema. Currently, the only possbile way is to manually add `$schema` property pointing to an online schema in every block. Local json schema or VsCode built-in settings for mapping schema do not work.

**Comment by kingyue737** [comment]
I've just seen someone request the same feature in vetur.
Is it possible to implement by volar plugins?

https://github.com/vuejs/vetur/discussions/2932

**Comment by johnsoncodehk** [comment]
Yes this is easy in my mind, you can open a feature request to https://github.com/johnsoncodehk/volar-plugins.
